src/style.css

```
@@ -1,3 +1,71 @@
 body {
   margin: 0;
 }
+
+:root {
+  --activ-color: #00acdc;
```

```suggestion
  --active-color: #00acdc;
```
Also, better call it `color-active` so all colors can be found easily among the variables